tests/bootstrap.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types = 1);
```

Comment:
Hast du die überall anders schon hinzugefügt gehabt und nur hier vergessen? Sonst würde ich diese Zeile in sehr vielen Dateien vermissen.